R/package-auth.R

```diff
@@ -152,18 +152,18 @@ read_databrickscfg <- function(key = c("token", "host", "wsid"), profile = NULL)
   # return error in case of empty profile
   if (is.null(vars)) {
     stop(cli::format_error(c(
-      "Specified {.var profile} not found in {.file `~/.databrickscfg`}:",
+      "Specified {.var profile} not found in {.file `{config_path}`}:",
       "x" = "Need to specify {.envvar {profile}} profile within {.file {config_path}} file."
     )))
   }
 
   # attempt to fetch required key & value pair from profile
   # error if key isn't found
   value <- vars[[key]]
-  if (is.null(vars)) {
+  if (is.null(value)) {
     stop(cli::format_error(c(
-      "Parameter {.var key} not found in profile of {.file {config_path}}:",
-      "x" = "Need to specify {.envvar {key}} in {.envvar {profile}} profile."
+      "Parameter {.var {key}} not found in profile of {.envvar {profile}}:",
+      "x" = "Need to specify {.envvar {key}} variable in {.envvar {profile}} profile within {.file {config_path}} file."
```

Comment:
Making error message more descriptive, so it is clear what needs to be done

Reply:
It should still be `{.file {config_path}}` as thats the headline error, it's missing in the profile of the config file.

This feels redundant if we say missing in config file (as above).
```
"Need to specify {.envvar {key}} variable in {.envvar {profile}} profile within {.file {config_path}} file."
```

I'd prefer either:
```
 stop(cli::format_error(c(
      "Parameter {.var {key}} not found in profile of {.file {config_path}}:",
      "x" = "Need to specify {.envvar {key}} in {.envvar {profile}} profile."
    )))
```

or the slightly redundant...

```
 stop(cli::format_error(c(
      "Parameter {.var {key}} not found in {.envvar {profile}} profile of {.file {config_path}}:",
      "x" = "Need to specify {.envvar {key}} in {.envvar {profile}} profile."
    )))
```

Reply:
I personally prefer detailed error messages so that it is clear what needs to be done where (if possible). But either option you suggested is fine by me.

Reply:
This has now been implemented